### PR TITLE
Add Oxford-EEBO-TCP translator

### PR DESCRIPTION
**Pull request by pierwill**
Not the prettiest or most flexible translator, but it's better than nothing for this important archive.

### CODE REVIEW

File: Oxford-EEBO-TCP.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+function doWeb(doc, url) {
+    
+    newItem = new Zotero.Item("book");
+    
+    if (url.includes("http://tei.it.ox.ac.uk/tcp/Texts-HTML/free/") == true) {
+	html(doc, url);
+    } else {
+	catalogue(doc, url);
+    }
```
Comment (zuphilip) [inline Oxford-EEBO-TCP.js]: The following scraping from that page looks very fragile. Moreover, there should be the corresponding TEI file available, e.g. http://tei.it.ox.ac.uk/tcp/Texts-TEI/free/A20/A20631.xml which has the information in structured form and we should therefore use instead.

Comment (zuphilip) [reply Oxford-EEBO-TCP.js]: Stopped here with the review therefore.

File: Oxford-EEBO-TCP.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
+{
+	"translatorID": "62151d94-6fdf-442f-be29-94bbd24fcd09",
+	"label": "Oxford-EEBO-TCP",
```
[the remark this answers is not in the document]
Comment (adam3smith) [reply Oxford-EEBO-TCP.js]: (EEBO = Early English Books Online -- there are different versions of this; Gale has one too, e.g.)

**Comment by pierwill** [comment]
Thanks for the look! I learned a lot writing this from scratch. Nice to finally start learning to contribute to something I use every day.

I'll look into parsing the TEI.

**Pull request closed without merging**